### PR TITLE
DM-39741: Update MTCS methods to handle m1m3 force balance system.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: python/lsst/ts/standardscripts/maintel/laser_tracker/align.py [hunk 1]
```diff
@@ -35,8 +35,8 @@
 
 # TODO (DM-38880) - Subsitute by class in idl.enum when available
```
Comment (edennihy) [inline python/lsst/ts/standardscripts/maintel/laser_tracker/align.py:36]: This ticket is merged, can you replace these with the AlignComponent class there?

File: Jenkinsfile [hunk 2]
```diff
@@ -2,6 +2,6 @@
 DevelopPipeline(
     name: "ts_standardscripts",
     module_name: "lsst.ts.standardscripts",
-    extra_packages: ["lsst-ts/ts_observatory_control"],
+    extra_packages: ["lsst-ts/ts_cRIOpy", "lsst-ts/ts_observatory_control"],
```
Comment (edennihy) [inline Jenkinsfile:5]: Should this be "lsst-ts/ts_criopy" instead? Not sure if this is case-sensitive here...

File: doc/version_history.rst [hunk 3]
```diff
@@ -6,6 +6,23 @@
...
+  * Skip alignment if tolerances are zero.
+  * Get last ``offsetPublished`` if new event is not available.
+  * Fix enum values.
+  * Skip error if laserTracker status is not available.
+
+* In ``system_wide_shutdown.py``, add more logging information.
+* Update ``tests/test_maintel_home_both_axes.py`` to check that force balance was disabled before homing.
+* In ``maintel/home_both_axes.py``, update execution to switch off force balance before homing.
+
+* Update Jenkinsfile to add ts_cRIOpy as an extra package.
```
Comment (edennihy) [inline doc/version_history.rst:23]: Also here and on the line below, package name should be updated to "ts_criopy"

**Review by edennihy — APPROVED** [review]
A few minor comments to consider, otherwise approved.

File: Jenkinsfile (hunk 2 above)
Comment (tribeiro) [reply Jenkinsfile:5]: No, this needs to be the name of the GitHub package, which is the `ts_cRIOpy` version.

File: doc/version_history.rst (hunk 3 above)
Comment (tribeiro) [reply doc/version_history.rst:23]: yeah, the name of the package is actually `ts_cRIOpy`... 🤷

File: python/lsst/ts/standardscripts/maintel/laser_tracker/align.py (hunk 1 above)
Comment (tribeiro) [reply python/lsst/ts/standardscripts/maintel/laser_tracker/align.py:36]: No, not yet.. only when the new cycle is released..

**Pull request merged**